### PR TITLE
Bl2494 longpress compatible with ckeditor

### PR DESCRIPTION
**Pull request by hatton**
Removed the arrow keys, as ckeditor was getting activated by them and messing up several things, including focus, which character got selected, and where it got inserted.

Added localizable instructions at the bottom of the longpress popup so people know what they are supposed to do.

### CODE REVIEW

File: src/BloomBrowserUI/lib/long-press/jquery.longpress.js [hunk 1]
```diff
@@ -112,26 +114,30 @@
...
     var storedOffset;
 
-    var popup=$('<ul class=long-press-popup />');
+    var popup;
 
     $(window).mousewheel(onWheel);
 
     function onKeyDown(e) {
-
+        /* we had to disable thes because ckeditor was seeing them and messing things up. Hopefully in the future it can be reinstated:
```
Comment (JohnThomson) [inline src/BloomBrowserUI/lib/long-press/jquery.longpress.js]: thes -> these

File: src/BloomBrowserUI/lib/long-press/jquery.longpress.js [hunk 2]
```diff
@@ -112,26 +114,30 @@
...
         // Arrow key with popup visible
-        if ($('.long-press-popup').length>0 && (e.which==37 || e.which==39)) {
-            if (e.which==37) activePreviousLetter();
+        if ($('.long-press-popup').length > 0 && (e.which == 37 || e.which == 39)) {
+            e.stopPropagation(); //stop ckeditor from seeing this (DOESN"T WORK)
+            e.preventDefault();
+            if (e.which == 37) activePreviousLetter();
```
Comment (JohnThomson) [inline src/BloomBrowserUI/lib/long-press/jquery.longpress.js]: Other stuff in PR suggests you meant to suppress the activatePrevious/Next behavior but they seem to be still present here.

File: src/BloomBrowserUI/lib/long-press/jquery.longpress.js [hunk 3]
```diff
@@ -164,13 +170,13 @@
         }
     }
     function showPopup(chars) {
-        popup.empty();
+        popup.find('ul').empty();
```
Comment (JohnThomson) [inline src/BloomBrowserUI/lib/long-press/jquery.longpress.js]: Don't see what this change is for...feels like we should comment and explain wherever we change a lib file

**Comment by JohnThomson** [comment]
In case we one day want to merge a new version of LongPress, should we do more to document what we changed?

File: src/BloomBrowserUI/lib/long-press/jquery.longpress.js (hunk 3 above)
Comment (hatton) [inline src/BloomBrowserUI/lib/long-press/jquery.longpress.js]: See line 9, which does that. But really, we have gone far from the original now, and would like to go a lot farther.

Comment (hatton) [inline src/BloomBrowserUI/lib/long-press/jquery.longpress.js]: See line 116/117 where the definition/structure of popup was changed to make room for instructions.

**Comment by hatton** [comment]
A diff tells you what changed, and the the commit record, along with section at the top (e.g. line 9) says why.

File: src/BloomBrowserUI/lib/long-press/jquery.longpress.js (hunk 1 above)
Comment (hatton) [inline src/BloomBrowserUI/lib/long-press/jquery.longpress.js]: fixed

File: src/BloomBrowserUI/lib/long-press/jquery.longpress.js (hunk 2 above)
Comment (hatton) [inline src/BloomBrowserUI/lib/long-press/jquery.longpress.js]: Github coloring is just letting you down :-) That section is commented out.

**Pull request merged**